app/stac_api/validators.py

```diff
@@ -49,6 +49,7 @@
     MediaType('application/vnd.laszip', 'Zipped LIDAR', ['.laz', '.zip']),
     MediaType('application/x.shapefile+zip', 'Zipped Shapefile', ['.zip']),
     MediaType('application/x.filegdb+zip', 'Zipped File Geodatabase', ['.zip']),
+    MediaType('application/x.filegdbp+zip', 'Zipped File Geodatabase', ['.zip']),
```

Comment:
I would use another description, I've no idea what is the difference between x.filegdb+zip and x.filegdbp+zip, but maybe ask @LucaSchneller or @phanuel for better description.

Reply:
Zipped File Geodatabase (ArcGIS Pro)